client/my-sites/plugins/plugin-details-CTA/index.jsx

```diff
@@ -288,7 +288,6 @@ const PluginDetailsCTA = ( { plugin, isPlaceholder } ) => {
 					<CTAButton
 						plugin={ plugin }
 						hasEligibilityMessages={ hasEligibilityMessages }
-						eligibilityHolds={ eligibilityHolds }
```

Comment:
Reverts part of this [earlier change](https://github.com/Automattic/wp-calypso/pull/68365/files?w=1#diff-84ecb17a0e2202226688ef5fdee0a57a431fc680dc4d9494783cc51ff84898baR291) as they're only needed for that title display logic which is now inside the warnings component.